assets/src/dashboard/app/views/exploreTemplates/content/index.js

```diff
@@ -91,16 +100,31 @@ function Content({
                 </>
               ) : (
                 <EmptyContentMessage>
-                  {search?.keyword
-                    ? sprintf(
-                        /* translators: %s: search term. */
-                        __(
-                          'Sorry, we couldn\'t find any results matching "%s"',
-                          'web-stories'
-                        ),
-                        search.keyword
-                      )
-                    : __('No templates currently available', 'web-stories')}
+                  <Headline
+                    size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+                    as="h3"
+                  >
+                    {search?.keyword
+                      ? sprintf(
+                          /* translators: %s: search term. */
+                          __(
+                            'Sorry, we couldn\'t find any results matching "%s"',
+                            'web-stories'
+                          ),
+                          search.keyword
+                        )
+                      : __('No templates currently available.', 'web-stories')}
+                  </Headline>
+                  {!search?.keyword && (
+                    <Button
+                      type={BUTTON_TYPES.PRIMARY}
+                      size={BUTTON_SIZES.MEDIUM}
+                      as="a"
+                      href={newStoryURL}
+                    >
+                      {__('Create a new Story', 'web-stories')}
+                    </Button>
+                  )}
```

Comment:
💭 Loading state:

I think we should show a loading state or something indicating that templates are loading to the user so that in the event of a slow connection we show them that things are happening in the background.

Reply:
yes! I thought there was a ticket for this. It's been brought up a few times now. couldn't find a ticket so i made one. https://github.com/google/web-stories-wp/issues/6622